synthtool/gcp/templates/python_library/docs/conf.py.j2

```diff
@@ -38,6 +38,7 @@ extensions = [
     "sphinx.ext.napoleon",
     "sphinx.ext.todo",
     "sphinx.ext.viewcode",
+    "recommonmark",
```

Comment:
We did the same change in python-storage: https://github.com/googleapis/python-storage/pull/103

Didn't realize it came from this parent template. The rationale for it:

https://github.com/readthedocs/recommonmark

```
# for Sphinx-1.4 or newer
extensions = ['recommonmark'] # <---- this 

# for Sphinx-1.3
from recommonmark.parser import CommonMarkParser

source_parsers = {
    '.md': CommonMarkParser,
}

source_suffix = ['.rst', '.md']
```